pwiz_tools/Skyline/Alerts/MsFraggerDownloadDlg.cs

```diff
@@ -0,0 +1,185 @@
+﻿/*
+ * Original author: Matt Chambers <matt.chambers42 .at. gmail.com >
+ *
+ * Copyright 2021 University of Washington - Seattle, WA
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Specialized;
+using System.Drawing;
+using System.IO;
+using System.Net;
+using System.Text.RegularExpressions;
+using System.Windows.Forms;
+using Ionic.Zip;
+using pwiz.Skyline.Controls;
+using pwiz.Skyline.Model.DdaSearch;
+using pwiz.Skyline.Properties;
+using pwiz.Skyline.Util;
+using pwiz.Skyline.Util.Extensions;
+
+namespace pwiz.Skyline.Alerts
+{
+    public partial class MsFraggerDownloadDlg : Form
+    {
+        private const string LICENSE_URL = @"https://msfragger.arsci.com/upgrader/MSFragger-LICENSE.pdf";
+        private const string DOWNLOAD_URL = @"http://msfragger-upgrader.nesvilab.org/upgrader/upgrade_download.php";
+        private const string DOWNLOAD_METHOD = @"POST";
+        private readonly Uri DOWNLOAD_URL_FOR_FUNCTIONAL_TESTS = new Uri($@"https://example.com/MSFragger-{MsFraggerSearchEngine.MSFRAGGER_VERSION}.zip");
+
+        private Tuple<int, int> licenseLinkStartEnd;
+
+        public MsFraggerDownloadDlg()
+        {
+            InitializeComponent();
+
+            const string BOLD_PATTERN = "<b>(.*?)</b>";

```

Comment:
@brendanx67 As long as the translators preserve the <b> and <link> tags, the highlighting and linking should work in the translated text too.